frontend/readhistory.lua

```diff
@@ -70,7 +70,7 @@ function ReadHistory:getIndexByTime(item_time)
     elseif item_time < self.hist[hist_nb].time then
         return hist_nb + 1
     end
-    -- binary search
+    -- binary search (returns min index when several elements equal item_time)
     local s, e, m, d = 1, hist_nb
```

Comment:
Line 76 can be optimized like
m = bit.rshift(e + s, 1)

And the elseif can be turned into an else in line 79.

Reply:
Thanks